less/navigation.less

```diff
@@ -1 +1,51 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+
+.header{
+    padding:2% 0;
+    border-bottom:@border;
+    
+    @media @mobile{
+        padding:0;
+    }
+
+    h1{
+        margin: 3% 0 2% 0;
+    }
+
+    div{
+        margin: 0 auto;
+        width:800px;
+        .flexbox();
+        justify-content: space-between;
+
+        @media @mobile{
+            margin: 0 auto;
```

Comment:
Inside of this media query:

- Because you have `margin` and `.flexbox();` outside of the media query, you don't have to retype it inside the media query.
- I would take `width: 100%;` outside the media query. Up above, I would type the two width properties like
```
max-width: 800px;
width: 100%;
```
The max-width will be 800px, but as the screen gets smaller, the nav bar will stay at the width of the screen.